### PR TITLE
Test rsgcn

### PR DESCRIPTION
**Pull request by corochann**
- Adding `dropout_ratio` attribute to control dropout ratio in RSGCN.
 - Added a test for RSGCN
    - forward
    - backward
    - graph isomorphism invariant

### CODE REVIEW

File: tests/models_tests/test_rsgcn.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+
+@pytest.fixture
+def data():
+    atom_data = numpy.random.randint(
+        0, high=MAX_ATOMIC_NUM, size=(batch_size, atom_size)
+    ).astype(numpy.int32)
+    # adj_data = numpy.random.randint(
+    #     0, high=2, size=(batch_size, atom_size, atom_size)
+    # ).astype(numpy.float32)
```
Comment (delta2323) [inline tests/models_tests/test_rsgcn.py]: Remove unused code.

File: tests/models_tests/test_rsgcn.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+    # adj_data is symmetric matrix
+    adj_data = numpy.random.uniform(
+        0, high=1, size=(batch_size, atom_size, atom_size)
+    ).astype(numpy.float32)
+    adj_data = adj_data + adj_data.swapaxes(-1, -2)
+    # adj_data = (adj_data > 1.5).astype(numpy.float32)
```
Comment (delta2323) [inline tests/models_tests/test_rsgcn.py]: Remove unused code.

**Comment by delta2323** [comment]
Regarding stochastic behavior of `dropout` in RSGCN, we can fix a mask by holding an instance of `F.Dropout` in RSGCN and reuse it in every call. We do not want such deterministic behavior in ordinal use cases, one way is to add an option to `RSGCN` to switch stochastic/deterministic behavior of `dropout`. See the test of Dropout in Chainer ([link](https://github.com/chainer/chainer/blob/master/tests/chainer_tests/functions_tests/noise_tests/test_dropout.py#L74)).

**Comment by corochann** [comment]
Is this change enough?

**Comment by delta2323** [comment]
Regarding my previous comment, it is OK to focus on deterministic behavior of dropout in another PR.

**Comment by corochann** [comment]
Could you merge if there is no request changes?

**Comment by delta2323** [comment]
Example tests passed.

**Comment by delta2323** [comment]
I created #153 for the dropout issue.

**Pull request merged**